askui_example/logging/logger.ts

```diff
@@ -1,10 +1,28 @@
 import "@askui/jest-allure-circus";
+import * as os from 'os';
 
 declare const allure: {
     step(name: string, body?: () => void): void;
     attachment(name: string, content: string, type: string): void;
+    writeEnvironmentInfo(info: Record<string, string>): void;
 }
```

Comment:
Is it not possible to import allure? Without type declaration? 

So that you  delete this:
```suggestion
import "@askui/jest-allure-circus";
import * as os from 'os';